sunbeam/commands/bootstrap.py

```diff
@@ -62,20 +67,27 @@
 @click.command()
 @click.option(
     "--role",
-    default="converged",
-    type=click.Choice(["control", "compute", "converged"], case_sensitive=False),
+    multiple=True,
+    default=["control", "compute"],
+    type=click.Choice(["control", "compute", "storage"], case_sensitive=False),
     help="Specify whether the node will be a control node, a "
-    "compute node, or a converged node (default)",
+    "compute node or a storage node. Defaults to all the roles.",
 )
 def bootstrap(role: str) -> None:
```

Comment:
This should become a list of str.

```suggestion
def bootstrap(roles: List[str]) -> None:
```

In a better version, it could also be (through the use of the click validation callback):
```suggestion
def bootstrap(roles: List[Role]) -> None:
```

Reply:
Done, good suggestion.